backend/src/indexer/block_preprocessor.rs

```diff
@@ -207,7 +210,7 @@ impl concordium_rust_sdk::indexer::Indexer for BlockPreProcessor {
                     RewardsOverview::V1 {
                         total_staked_capital,
                         ..
-                    } => *total_staked_capital,
+                    } => (*total_staked_capital, HashMap::new()),
```

Comment:
We would want to fetch the baker stake here as well, otherwise no stake changes starting from `RewardsOverview::V1`  (P4 I think)